alchemist-full/build.gradle.kts

```diff
@@ -145,6 +146,43 @@ tasks.register<Exec>("testJpackageOutput") {
     finalizedBy(deleteJpackageOutput)
 }
 
+val generatePKGBUILD by tasks.registering(Exec::class) {
```

Comment:
```suggestion
val generatePKGBUILD by tasks.registering<Exec> {
```

Reply:
The `<Exec>` version gives me an error.
After looking in the documentation I found tasks configured like this:
`val {name} by tasks.registering(Exec::class) {`
or 
`tasks.register<Exec>("{name}") {`